index 320a375..d937631 100644
--- a/Makefile
+++ b/Makefile
@@ -414,3 +414,13 @@ generate-files:
 	python3 scripts/generate_functions.py
 	python3 scripts/generate_serialization.py
 	python3 scripts/generate_enum_util.py
+
+bundle-library: release
+	cd build/release && \
+	mkdir -p bundle && \
+	cp src/libduckdb_static.a bundle/. && \
+	cp third_party/*/libduckdb_*.a bundle/. && \
+	cp extension/*/lib*_extension.a bundle/. && \
+	cd bundle && \
+	find . -name '*.a' -exec ${AR} -x {} \; && \
+	${AR} cr ../libduckdb_bundle.a *.o
